### PR TITLE
Glist implementing

### PR DESCRIPTION
**Pull request by ghost**
Making glist ban and unban shows reason as public and the ip as client-message.

Requested by alot of admins because some ppl after glist banned reported on forum Some DDOSS attack happening...

### CODE REVIEW

**Comment by ItzLevvie** [comment]
Some incorrect formatting...

**Comment by Wild1145** [comment]
Thank you for contributing to the project, could you please amend your opening message to include the issue that this PR resolves. Thank You.

**Comment by marcocorriero** [comment]
#1726

**Comment by Wild1145** [comment]
This PR will remain on hold until the associated issue is reviewed, discussed and approved.

**Comment by marcocorriero** [comment]
#1834

**Comment by ghost** [comment]
Bump, @JeromSar

**Comment by ItzLevvie** [comment]
@marcocorriero Read my comment above

**Comment by ghost** [comment]
@OxLemonxO im not good at coding, you can do me a favour fixing this code?

**Comment by Wild1145** [comment]
This PR will remain on hold until the associated issue is reviewed, discussed and approved.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java:19]: Formatting needs changing here.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java:22]: Formatting also needs changing here.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java:23]: Formatting also needs changing here.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java:31]: Formatting is off here.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_glist.java:32]: Formatting needs changing here.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please address the requested changes before this PR can be merged.

**Comment by ghost** [comment]
@OxLemonxO maybe can you fix this code? 
i didnt understand what @Wild1145 is asking

**Comment by ItzLevvie** [comment]
Your code has to be in Allman Style... not K&R

**Comment by ghost** [comment]
@Wild1145 The branch for this PR has been deleted, it is now invalid.

**Comment by Wild1145** [comment]
Due to the original repo having been destroyed, I am closing this PR down as a result.

**Comment by JeromSar** [comment]
@Wild1145 The host repo's deletion doesn't make this an invalid PR though. An invalid PR is a PR with illogical changes and/or no sign of any reason on which to base that the author understands what he/she is doing.

**Comment by Wild1145** [comment]
@JeromSar It is however invalid in the regards that the PR is not valid, as it could not be merged? If thats not how you're categorising "Invalid" it would be worth looking to make additional labels that better categorise such behaviours.

**Comment by JeromSar** [comment]
@Wild1145 Maybe something for the future, we would have to relabel previously unmerged PRs too.

**Comment by Wild1145** [comment]
@JeromSar Only if the repo was deleted, anything that's not merged I have been tagging as PR:Declined but anything I physically couldnt merge for whatever reason and is not being fixed, I've tagged invalid as its not possible to merge even if we wanted to.

**Pull request closed without merging**
